pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test: TestLogic_mixed_version_stats failed

pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test.TestLogic_mixed_version_stats [failed](https://mesolite.cluster.engflow.com/invocations/default/255c5e49-523a-4a82-b449-e66fb829a26b?testReportRun=1&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvc3FsL2xvZ2ljdGVzdC90ZXN0cy9jb2Nrcm9hY2gtZ28tdGVzdHNlcnZlci0yNC4zOmNvY2tyb2FjaC1nby10ZXN0c2VydmVyLTI0XzNfdGVzdA==) on master @ [64220c2113c95e01f29afee9e7338e14ecc04ee6](https://github.com/cockroachdb/cockroach/commits/64220c2113c95e01f29afee9e7338e14ecc04ee6):


```
=== RUN   TestLogic_mixed_version_stats
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogic_mixed_version_stats404280766
    test_log_scope.go:76: use -show-logs to present logs inline
[20:49:20] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 1 statements
[20:49:22] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 2 statements
[20:49:50] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 3 statements
[20:49:52] --- done: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats with config cockroach-go-testserver-24.3: 5 tests, 0 failures
    logic.go:4336: 
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: error while processing
    logic.go:4336: 
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: 
        expected success, but found
        dial tcp 127.0.0.1:38349: connect: connection refused
cockroach logs captured in: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/testlogs/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test/shard_8_of_11/test.outputs/cockroach-logs2831943271
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestLogic_mixed_version_stats404280766
--- FAIL: TestLogic_mixed_version_stats (43.40s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_mixed_version_stats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-50401

pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test.TestLogic_mixed_version_stats [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19624084?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19624084?buildTab=artifacts#/) on master @ [64220c2113c95e01f29afee9e7338e14ecc04ee6](https://github.com/cockroachdb/cockroach/commits/64220c2113c95e01f29afee9e7338e14ecc04ee6):


```
=== RUN   TestLogic_mixed_version_stats
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats3154703235
    test_log_scope.go:76: use -show-logs to present logs inline
[21:48:07] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/32881/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 1 statements
[21:48:09] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/32881/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats with config cockroach-go-testserver-24.3: 5 tests, 0 failures
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/32881/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: error while processing
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/32881/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: 
        expected success, but found
        dial tcp 127.0.0.1:36217: connect: connection refused
cockroach logs captured in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/cockroach-logs2786639608
    panic.go:635: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats3154703235
--- FAIL: TestLogic_mixed_version_stats (13.48s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_mixed_version_stats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test.TestLogic_mixed_version_stats [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19625176?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19625176?buildTab=artifacts#/) on master @ [7d1a0ecfea73a5c45378036881fe106cdb2b35eb](https://github.com/cockroachdb/cockroach/commits/7d1a0ecfea73a5c45378036881fe106cdb2b35eb):


```
=== RUN   TestLogic_mixed_version_stats
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats1085950473
    test_log_scope.go:76: use -show-logs to present logs inline
[22:33:13] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/27838/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 1 statements
[22:33:15] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/27838/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 3 statements
[22:33:17] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/27838/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats with config cockroach-go-testserver-24.3: 5 tests, 0 failures
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/27838/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: error while processing
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/27838/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: 
        expected success, but found
        dial tcp 127.0.0.1:36481: connect: connection refused
cockroach logs captured in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/cockroach-logs3664312297
    panic.go:635: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats1085950473
--- FAIL: TestLogic_mixed_version_stats (16.07s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_mixed_version_stats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test.TestLogic_mixed_version_stats [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19625515?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19625515?buildTab=artifacts#/) on master @ [7bef357d9d15b0d09638a4b36f3c20c4118311af](https://github.com/cockroachdb/cockroach/commits/7bef357d9d15b0d09638a4b36f3c20c4118311af):


```
=== RUN   TestLogic_mixed_version_stats
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats3816689854
    test_log_scope.go:76: use -show-logs to present logs inline
[23:23:56] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/20694/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 1 statements
[23:24:26] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/20694/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 3 statements
[23:24:28] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/20694/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats with config cockroach-go-testserver-24.3: 5 tests, 0 failures
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/20694/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: error while processing
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/20694/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: 
        expected success, but found
        read tcp 127.0.0.1:42154->127.0.0.1:38427: read: connection reset by peer
cockroach logs captured in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/cockroach-logs1250487965
    panic.go:635: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats3816689854
--- FAIL: TestLogic_mixed_version_stats (43.27s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_mixed_version_stats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test.TestLogic_mixed_version_stats [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19626000?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19626000?buildTab=artifacts#/) on master @ [8d775034b09219c723a94e96b918ed258aa36c50](https://github.com/cockroachdb/cockroach/commits/8d775034b09219c723a94e96b918ed258aa36c50):


```
=== RUN   TestLogic_mixed_version_stats
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats2220202854
    test_log_scope.go:76: use -show-logs to present logs inline
[00:55:27] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31008/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 1 statements
[00:55:58] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31008/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 3 statements
[00:55:59] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31008/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats with config cockroach-go-testserver-24.3: 5 tests, 0 failures
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31008/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: error while processing
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31008/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: 
        expected success, but found
        read tcp 127.0.0.1:39656->127.0.0.1:45985: read: connection reset by peer
cockroach logs captured in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/cockroach-logs416805116
    panic.go:635: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats2220202854
--- FAIL: TestLogic_mixed_version_stats (43.09s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_mixed_version_stats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test.TestLogic_mixed_version_stats [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19626236?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19626236?buildTab=artifacts#/) on master @ [de844314fcad511ba76f5eafd40928b1de2822fc](https://github.com/cockroachdb/cockroach/commits/de844314fcad511ba76f5eafd40928b1de2822fc):


```
=== RUN   TestLogic_mixed_version_stats
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats2565051660
    test_log_scope.go:76: use -show-logs to present logs inline
[02:21:50] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/20645/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 1 statements
[02:21:52] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/20645/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats with config cockroach-go-testserver-24.3: 5 tests, 0 failures
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/20645/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: error while processing
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/20645/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: 
        expected success, but found
        dial tcp 127.0.0.1:41135: connect: connection refused
cockroach logs captured in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/cockroach-logs2276867846
    panic.go:635: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats2565051660
--- FAIL: TestLogic_mixed_version_stats (13.34s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_mixed_version_stats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test.TestLogic_mixed_version_stats [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19626761?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19626761?buildTab=artifacts#/) on master @ [c4638a99e2add36405ada00dc82fca470403ae58](https://github.com/cockroachdb/cockroach/commits/c4638a99e2add36405ada00dc82fca470403ae58):


```
=== RUN   TestLogic_mixed_version_stats
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats1183498039
    test_log_scope.go:76: use -show-logs to present logs inline
[03:57:25] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/663/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 1 statements
[03:57:55] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/663/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 3 statements
[03:57:57] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/663/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats with config cockroach-go-testserver-24.3: 5 tests, 0 failures
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/663/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: error while processing
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/663/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: 
        expected success, but found
        read tcp 127.0.0.1:33402->127.0.0.1:44463: read: connection reset by peer
cockroach logs captured in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/cockroach-logs1489700906
    panic.go:635: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats1183498039
--- FAIL: TestLogic_mixed_version_stats (43.22s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_mixed_version_stats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test.TestLogic_mixed_version_stats [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19626709?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19626709?buildTab=artifacts#/) on master @ [a710718c1cd9590aa124b69d7672166038eacf9b](https://github.com/cockroachdb/cockroach/commits/a710718c1cd9590aa124b69d7672166038eacf9b):


```
=== RUN   TestLogic_mixed_version_stats
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats4120223355
    test_log_scope.go:76: use -show-logs to present logs inline
[04:33:44] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19865/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 1 statements
[04:34:15] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19865/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 3 statements
[04:34:16] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19865/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats with config cockroach-go-testserver-24.3: 5 tests, 0 failures
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19865/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: error while processing
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19865/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: 
        expected success, but found
        dial tcp 127.0.0.1:38977: connect: connection refused
cockroach logs captured in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/cockroach-logs3392813060
    panic.go:635: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats4120223355
--- FAIL: TestLogic_mixed_version_stats (43.26s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_mixed_version_stats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test.TestLogic_mixed_version_stats [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19627070?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19627070?buildTab=artifacts#/) on master @ [c4638a99e2add36405ada00dc82fca470403ae58](https://github.com/cockroachdb/cockroach/commits/c4638a99e2add36405ada00dc82fca470403ae58):


```
=== RUN   TestLogic_mixed_version_stats
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats902844143
    test_log_scope.go:76: use -show-logs to present logs inline
[05:49:23] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/27949/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 1 statements
[05:49:24] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/27949/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats with config cockroach-go-testserver-24.3: 5 tests, 0 failures
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/27949/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: error while processing
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/27949/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: 
        expected success, but found
        dial tcp 127.0.0.1:39685: connect: connection refused
cockroach logs captured in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/cockroach-logs2098539973
    panic.go:635: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats902844143
--- FAIL: TestLogic_mixed_version_stats (13.13s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_mixed_version_stats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test.TestLogic_mixed_version_stats [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19627400?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19627400?buildTab=artifacts#/) on master @ [561a43d20ce870dbe0ad12fc3e51e3bee49c60e7](https://github.com/cockroachdb/cockroach/commits/561a43d20ce870dbe0ad12fc3e51e3bee49c60e7):


```
=== RUN   TestLogic_mixed_version_stats
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats2741185953
    test_log_scope.go:76: use -show-logs to present logs inline
[06:12:08] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/32432/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 1 statements
[06:12:10] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/32432/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats with config cockroach-go-testserver-24.3: 5 tests, 0 failures
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/32432/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: error while processing
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/32432/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: 
        expected success, but found
        dial tcp 127.0.0.1:39203: connect: connection refused
cockroach logs captured in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/cockroach-logs3287927662
    panic.go:635: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats2741185953
--- FAIL: TestLogic_mixed_version_stats (13.32s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_mixed_version_stats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test.TestLogic_mixed_version_stats [failed](https://tanzanite.cluster.engflow.com/invocations/default/3820c677-7ab7-4451-a38e-f4f599c9f319?testReportRun=1&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvc3FsL2xvZ2ljdGVzdC90ZXN0cy9jb2Nrcm9hY2gtZ28tdGVzdHNlcnZlci0yNC4zOmNvY2tyb2FjaC1nby10ZXN0c2VydmVyLTI0XzNfdGVzdA==) on master @ [c4638a99e2add36405ada00dc82fca470403ae58](https://github.com/cockroachdb/cockroach/commits/c4638a99e2add36405ada00dc82fca470403ae58):


```
=== RUN   TestLogic_mixed_version_stats
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogic_mixed_version_stats1543516297
    test_log_scope.go:76: use -show-logs to present logs inline
[05:07:38] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 1 statements
[05:07:41] --- done: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats with config cockroach-go-testserver-24.3: 5 tests, 0 failures
    logic.go:4336: 
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: error while processing
    logic.go:4336: 
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: 
        expected success, but found
        read tcp 127.0.0.1:60960->127.0.0.1:46679: read: connection reset by peer
cockroach logs captured in: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/testlogs/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test/shard_8_of_11_run_1_of_10/test.outputs/cockroach-logs1442825120
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestLogic_mixed_version_stats1543516297
--- FAIL: TestLogic_mixed_version_stats (13.34s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_mixed_version_stats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test.TestLogic_mixed_version_stats [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19629999?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19629999?buildTab=artifacts#/) on master @ [5fa10b572756b44b66c8d363989b414ae488bc23](https://github.com/cockroachdb/cockroach/commits/5fa10b572756b44b66c8d363989b414ae488bc23):


```
=== RUN   TestLogic_mixed_version_stats
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats313487619
    test_log_scope.go:76: use -show-logs to present logs inline
[15:56:26] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/28606/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 1 statements
[15:56:27] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/28606/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats with config cockroach-go-testserver-24.3: 5 tests, 0 failures
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/28606/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: error while processing
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/28606/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: 
        expected success, but found
        read tcp 127.0.0.1:44826->127.0.0.1:42407: read: connection reset by peer
cockroach logs captured in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/cockroach-logs2952122763
    panic.go:635: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats313487619
--- FAIL: TestLogic_mixed_version_stats (13.28s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_mixed_version_stats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test.TestLogic_mixed_version_stats [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19630317?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19630317?buildTab=artifacts#/) on master @ [a83af6fa563f98d07baf70809024583674de7414](https://github.com/cockroachdb/cockroach/commits/a83af6fa563f98d07baf70809024583674de7414):


```
=== RUN   TestLogic_mixed_version_stats
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats836914856
    test_log_scope.go:76: use -show-logs to present logs inline
[16:13:18] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31908/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 1 statements
[16:13:30] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31908/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 3 statements
[16:13:31] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31908/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats with config cockroach-go-testserver-24.3: 5 tests, 0 failures
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31908/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: error while processing
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31908/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: 
        expected success, but found
        dial tcp 127.0.0.1:33519: connect: connection refused
cockroach logs captured in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/cockroach-logs968329533
    panic.go:635: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats836914856
--- FAIL: TestLogic_mixed_version_stats (24.99s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_mixed_version_stats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test.TestLogic_mixed_version_stats [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19630557?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19630557?buildTab=artifacts#/) on master @ [4be35ba8ca46898778a15c68ec8ff5f90725089e](https://github.com/cockroachdb/cockroach/commits/4be35ba8ca46898778a15c68ec8ff5f90725089e):


```
=== RUN   TestLogic_mixed_version_stats
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats3060479885
    test_log_scope.go:76: use -show-logs to present logs inline
[16:51:42] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2805/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 1 statements
[16:52:13] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2805/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats: 3 statements
[16:52:14] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2805/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats with config cockroach-go-testserver-24.3: 5 tests, 0 failures
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2805/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: error while processing
    logic.go:4336: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2805/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/cockroach-go-testserver-24_3_test_/cockroach-go-testserver-24_3_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/mixed_version_stats:31: 
        expected success, but found
        read tcp 127.0.0.1:53366->127.0.0.1:41239: read: connection reset by peer
cockroach logs captured in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/cockroach-logs2560587000
    panic.go:635: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/298f64d1f1e27c58ef8125e2dd93c20b/logTestLogic_mixed_version_stats3060479885
--- FAIL: TestLogic_mixed_version_stats (43.28s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_mixed_version_stats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/sql/logictest/logictestbase/logictestbase.go
+++ b/pkg/sql/logictest/logictestbase/logictestbase.go
@@ -649,6 +649,8 @@ var DefaultConfigSets = map[string]ConfigSet{
 
 	// Special alias for all testserver configs (for mixed-version testing).
 	"cockroach-go-testserver-configs": makeConfigSet(
+		// TODO(yuzefovich): when removing 24.3 version from here, ensure to
+		// use the alias in mixed_version_stats.
 		"cockroach-go-testserver-24.3",
 		"cockroach-go-testserver-25.1",
 	),

--- a/pkg/sql/logictest/testdata/logic_test/mixed_version_stats
+++ b/pkg/sql/logictest/testdata/logic_test/mixed_version_stats
@@ -1,4 +1,4 @@
-# LogicTest: cockroach-go-testserver-configs
+# LogicTest: cockroach-go-testserver-25.1
 
 # This is a regression test for #137386 which verifies the hyperloglog library
 # upgrade for the table stats. We want such a distributed table stats collection

--- a/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/BUILD.bazel
+++ b/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/BUILD.bazel
@@ -11,7 +11,7 @@ go_test(
         "//pkg/sql/logictest:testdata",  # keep
     ],
     exec_properties = {"test.Pool": "heavy"},
-    shard_count = 11,
+    shard_count = 10,
     tags = ["cpu:3"],
     deps = [
         "//pkg/base",

--- a/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/generated_test.go
+++ b/pkg/sql/logictest/tests/cockroach-go-testserver-24.3/generated_test.go
@@ -115,13 +115,6 @@ func TestLogic_mixed_version_plpgsql_srf(
 	runLogicTest(t, "mixed_version_plpgsql_srf")
 }
 
-func TestLogic_mixed_version_stats(
-	t *testing.T,
-) {
-	defer leaktest.AfterTest(t)()
-	runLogicTest(t, "mixed_version_stats")
-}
-
 func TestLogic_mixed_version_timeseries_range_already_exists(
 	t *testing.T,
 ) {

